tslint.json

```diff
@@ -1,15 +1,18 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-        "tslint:recommended"
-    ],
-    "jsRules": {},
-    "rules": {
-        "callable-types": false,
-        "space-before-function-paren": false,
-        "interface-name": false
-    },
-    "rulesDirectory": [
-        "node_modules/tslint-microsoft-contrib/"
-    ]
-}
\ No newline at end of file
+  "extends": "tslint:latest",
+  "rules": {
+    "quotemark": [true, "single"],
+    "object-literal-sort-keys": false,
+    "ordered-imports": [false],
+    "whitespace": [true, "check-branch", "check-decl", "check-operator", "check-module", "check-separator", "check-type"],
+    "interface-name": [true, "never-prefix"],
+    "no-shadowed-variable": false,
+    "no-string-literal": false,
+    "trailing-comma": [false],
+    "member-ordering": [true, {"order": "fields-first"}],
+    "array-type": [true, "array-simple"],
+    "arrow-parens": false,
+    "max-classes-per-file": [false],
+    "prefer-for-of": false
+  }
+}
```

Comment:
What's the source of this set of tslint rules? For instance, is it the canonical ruleset for Aurelia TypeScript projects?

Reply:
[aurelia-ux](https://github.com/aurelia/ux/blob/master/tslint.json)